.github/workflows/test-plugin.yml

```diff
@@ -97,3 +97,88 @@ jobs:
     - name: Run CPU backend examples
       working-directory: triton/third_party/triton_shared/python/examples
       run: pytest .
+
+
+
+  build_and_test_triton_shared_arm:
+    runs-on: ["self-hosted", "1ES.Pool=triton-shared-github-arm"]
+
+    steps:
+    - name: Check Architecture
+      run: uname -a
+
+    - name: Force Failure
+      if: ${{ inputs.force-failure }}
+      run: exit 1
+
+    - name: Checkout Triton
+      uses: actions/checkout@v4
+      with:
+        repository: 'openai/triton'
+        ref: ${{ inputs.triton-ref }}
+        path: triton
+        submodules: 'recursive'
+
+    - name: Checkout Triton-Shared
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.triton-shared-ref }}
+        path: triton/third_party/triton_shared
+
+    - name: Clear Triton Cache
+      run: |
+        rm -rf ~/.triton
+
+    - name: install pip
+      run: |
+        wget https://bootstrap.pypa.io/get-pip.py
+        python3 get-pip.py --user
+        rm get-pip.py
+  
+    - name: Update PATH
+      run: |
+        echo "PATH=${HOME}/.local/bin:${PATH}" >> "${GITHUB_ENV}"
+
+    - name: Check pre-commit
+      run: |
+        cd triton
+        python3 -m pip install --upgrade pre-commit
+        python3 -m pre_commit run --all-files --verbose
+
+    - name: Build/Install Triton
+      run: |
+        export TRITON_CODEGEN_TRITON_SHARED=1
+        cd triton/python
+        python3 -m pip install --upgrade pip
+        python3 -m pip install cmake==3.24
+        python3 -m pip install ninja
+        python3 -m pip uninstall -y triton
+        rm setup.py 
```

Comment:
is the plan to get `setup.py` updated to handle ARM?

Reply:
this was a workaround for when my triton didn't support ARM, this can be removed when this repo is upto date with the latest build of triton